src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-version.html

```diff
@@ -0,0 +1,7 @@
+<p class="theme-version">
+    {{ _("Based on") }}
+    <a href="https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html">
+        Pydata-sphinx-theme
+    </a>
+    {{ theme_version }}.
```

Comment:
Could we put this in a little span element in case people want to style it?

Reply:
the whole thing is encapsulated in a `p` element with the `theme-version` class isn't it sufficient?

Reply:
If you prefer not to have the span, it's fine w me. I tend to wrap everything that is "semantically different" from anything else in its own class but that is maybe overkill 😅

In the future if somebody really wants it they will ask for it and we can consider adding it in

Reply:
I don't have a strong opinion on this one, I simply copied the structure of the `sphinx-version.html` template. 
If you want I can add a span to both or keep it as it is